src/component/tooltip/TooltipView.ts

```diff
@@ -528,8 +533,27 @@ class TooltipView extends ComponentView {
 
                 zrUtil.each(item.seriesDataIndices, function (idxItem) {
                     const series = ecModel.getSeriesByIndex(idxItem.seriesIndex);
+                    const data = series.getData();
                     const dataIndex = idxItem.dataIndexInside;
+                    const dims = zrUtil.map(series.coordinateSystem.dimensions, function (coordDim) {
+                        return data.mapDimension(coordDim);
+                    });
                     const dataParams = series && series.getDataParams(dataIndex) as TooltipDataParams;
+                    let isStacked = false;
+                    const stackResultDim = data.getCalculationInfo('stackResultDimension');
+                    if (isDimensionStacked(data, dims[0])) {
+                        isStacked = true;
+                        dims[0] = stackResultDim;
+                    }
+                    if (isDimensionStacked(data, dims[1])) {
+                        isStacked = true;
+                        dims[1] = stackResultDim;
+                    }
+                    dataParams.position = findPointFromSeries({
```

Comment:
`dataParams` will be exposed to users in callback.
So every properties on `dataParams` should have a clear meaning for users, and any change in future will be a break change.
I think it's not easy to tell users what this `position`. And it might be changed in future. 
So it should better not be mounted on dataParams.